application/core/ASPA_Controller.php

```diff
@@ -16,13 +16,28 @@ class ASPA_Controller extends CI_Controller
      * @var array All the information for this event (retrieved from google sheet).
      */
     protected $eventData;
-
+    protected $org;
     function __construct()
     {
         parent::__construct();
         // $this->load->helper();
 		// $this->load->model();
         $this->eventData = $this->loadEventData();
+        $this->org = $this->loadOrganisationData();
+    }
+
+    /**
+     * This function handles the loading the Organisation model
+     *
+     * @return array
+     */
+    public function loadOrganisationData(){
+
+        $organisation = [];
+        $this->org = new Organisation_Model("Sanggy", "0123", "032", "30", "0", "-", "uoapool@gmail.com");
+        $elements = ['orgName', 'orgBankAccNumber', 'orgId', 'orgBankRefFormat', 'orgLogoImg', 'orgTagline', 'orgSupportEmail'];
+        $organisation = $this->org->array_Parameters($elements);
```

Comment:
Can we have the elements declared inside the method `array_Parameters` instead of having the references outside here?